src/locale/id.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   'email': 'Format email tidak valid.',
   'emptyString': 'Input harus kosong',
   'equal:$1': 'Input tidak sesuai.',
+  'equal-to-field:$1': 'Input tidak sesuai.',
```

Comment:
Shouldn't this `Input harus sama dengan <%= inputObj[ruleParams[0]] %>` ?

Reply:
done